### ISSUE TITLE
Unable to set tab or focus order on Shell flyout menu items

### ISSUE THREAD

**Issue by davidortinau** [xamarin/Xamarin.Forms#5131]
### Description
`ShellItem` is the current type that is placed in the Flyout. It doesn't have a way to set tab index or focus index. It does have AutomationProperties
Parent: #2415

**Comment by samhouts** [comment]
closed by #5930
